apps/client/src/app/components/position/position-detail-dialog/position-detail-dialog.html

```diff
@@ -213,15 +213,15 @@
       </ng-container>
     </div>
 
-    <div *ngIf="orders?.length > 0" class="row">
+    <div class="row" [ngClass]="{ 'd-none': !orders?.length }">
```

Comment:
In impersonation mode you have indeed no orders, that's why the table needs to be hidden (but not with `*ngIf` because this would break `mat-sort`)